common-codegen/build.gradle

```diff
@@ -1,6 +1,41 @@
+plugins {
+    id "net.ltgt.apt-idea" version "0.15"
+}
 
 dependencies {
     compile "io.vrap.rmf:raml-model:${versions.rmf}"
 
+    compile "org.immutables:value:2.6.1"
+    apt "org.immutables:value:2.6.1"
+    compile 'com.google.dagger:dagger:2.16'
+    apt 'com.google.dagger:dagger-compiler:2.16'
+    compile 'com.squareup:javapoet:1.10.0'
```

Comment:
I would prefer to use the syntax for dependencies everywhere, would be great if you could fix this 😄

Reply:
sorry didn't quite get the point here?

Reply:
Instead of a mix like `compile group: 'org.slf4j', name: 'slf4j-simple', version: '1.7.25'` I would prefer to have all dependencies in the uniform `compile 'org.slf4j:slf4j-simple:1.7.25'` form. Kind of signs of Obsessive Compulsive Disorder on my side 😄

Reply:
ah yeah  you are totally correct 😆  , it looks like like a Frankenstein code there 😅